WooCommerce/Classes/ViewRelated/Upgrades/WooPlanFeatureBenefitsView.swift

```diff
@@ -35,6 +35,10 @@ struct WooPlanFeatureBenefitsView: View {
         }
         .navigationTitle(wooPlanFeatureGroup.title)
         .navigationBarTitleDisplayMode(.inline)
+        .onAppear {
+            ServiceLocator.analytics.track(event:
+                    .InAppPurchases.planUpgradeFeatureScreenLoaded(featureGroup: wooPlanFeatureGroup.title))
```

Comment:
I'm not entirely positive if this is the best way to pass the featureGroup parameter here. Initially I created an enum on `WooAnalyticsEvent.InAppPurchases` with the different cases, but filtering it depending on the view we're loading seems to be trickier than I initially expected. Happy to iterate further if you think we should pass other than the `wooPlanFeatureGroup.title`